### ISSUE TITLE
Doesn't work with mongo db :(

### ISSUE THREAD

**Issue by Cryde** [tetranz/select2entity-bundle#45]
Hi,

It seems that your bundle doesn't work with mongodb : 
```
Type error: Argument 1 passed to Tetranz\Select2EntityBundle\Form\Type\Select2EntityType::__construct() must implement interface Doctrine\ORM\EntityManagerInterface, instance of Doctrine\ODM\MongoDB\DocumentManager given, called in
```
Basically MongoDB part of Doctrine doens't implement the same interface as "Entity" one ...

Any solution or workaround?

Thanks!

**Comment by Cryde** [comment]
I think that you should type with "ObjectManager" not with "EntityManagerInterface" to be able to work with ODM ...

**Comment by tetranz** [comment]
I've merged Cryde's pull request and released version 2.8 so it should work now.
